docs/releases.md

```diff
@@ -278,7 +278,11 @@ Adjust state and `Released As` of [Jira issues](https://issues.jenkins.io/) fixe
 ⚠️ **Requires write access to the [jenkinsci/docker repository](https://github.com/jenkinsci/docker) ** ⚠️
 💡 **Access to the network containing trusted.ci.jenkins.io could help in case of error but is not mandatory** 💡 
 
-Create a new tag in the [jenkinsci/docker repository](https://github.com/jenkinsci/docker) with the value of the released core version.
+Create a new tag in the [jenkinsci/docker repository](https://github.com/jenkinsci/docker) with the value of the released core version:
+- `git tag -am "2.VVV.V" 2.VVV.V`
```

Comment:
```suggestion
- `git tag --annotate --message="2.VVV.V" 2.VVV.V`
```

nit, reference at https://git-scm.com/docs/git-tag.
Personnal preference (non blocking at all!) to make the command easier to understand and for easier self-checks